services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -374,10 +424,15 @@ def compute_descriptive_statistics_response(
         for feature_name, feature in features.items()
         if isinstance(feature, dict) and feature.get("_type") == "Value" and feature.get("dtype") in NUMERICAL_DTYPES
     }
-    if not categorical_features and not numerical_features:
+    string_features = {
+        feature_name: feature
+        for feature_name, feature in features.items()
+        if isinstance(feature, dict) and feature.get("_type") == "Value" and feature.get("dtype") in STRING_DTYPES
+    }
+    if not categorical_features and not numerical_features and not string_features:
         raise NoSupportedFeaturesError(
             "No columns for statistics computation found. Currently supported feature types are: "
-            f"{NUMERICAL_DTYPES} and ClassLabel. "
+            f"{NUMERICAL_DTYPES}, {STRING_DTYPES} and ClassLabel. "
         )
 
     con = duckdb.connect(":memory:")  # we don't load data in local db file, use local parquet file instead
```

Comment:
do we have to free the memory at the end of the job by deleting the table that was created ?

Reply:
actually everything is cleared when con.close() is called so we're all good

Reply:
no, since it's in-memory, it's freed automatically when connection is closed.